src/revalue/voi_calculators.py

```
@@ -0,0 +1,266 @@
+        raise ValueError("recall_target must be between 0 and 1.")
+
+    # Create cumulative counts. Assumes results_df is ordered by screening.
+    # The index (0 to N-1) + 1 can serve as 'n_screened'.
+    # Make a copy to avoid SettingWithCopyWarning if results_df is a slice
+    results_df_copy = results_df.copy()
+    results_df_copy['n_screened'] = np.arange(1, len(results_df_copy) + 1)
+    results_df_copy['cumulative_relevant_found'] = results_df_copy['label'].cumsum()
+
+    target_relevant_items_to_find = recall_target * total_relevant_in_dataset
+    if target_relevant_items_to_find == 0: # Should be covered by recall_target == 0
+         return 100.0
```

Flake8 (7.2.0)

[error] indentation is not a multiple of 4

(E111)